hubzero/hubzero-cms
# [PURR][#2525]Add language to tiles on publication start page

https://purr.purdue.edu/support/ticket/2525

We want to add language to the publication tiles on publication start page, where the language asks researchers to reach out to hub administrator before choosing the database publication or series publication. The reason that we do it is that we see that a lot of researchers are not aware of the differences among file, database and series publication. The language will save a lot work and resource if researchers can talk with us first before selecting the publication type for their datasets.

The code changes include languages added to the publication tiles on start page where users choose the type of publication. There is a mailto link to the contact email address in the language. The contact email address can be set in the publication component configuration user interface through administrator interface. The language will display when both contact email address and doi publisher are set in publication component configuration on administrator user interface.

Test:
1. Set your hub's contact email in Contact Email Address field on publication component configuration through administrator interface. You also need to make sure that DOI Publisher is not empty in the publication component configuration on administrator interface.
2. Go to the publications in a project, click on "Start a new publication".
3. On the publication start page, the file publication tile should show the language "This is typically the best option for most PURR researchers.". 
The database tile should show the language "If you consider this type, please contact <a href='mailto:'Your hub's contact email address'?subject=Database publication'>Your hub's name administrator</a> first.". The series publication tile should show "If you consider this type, please contact <a href='mailto:'Your hub's contact email address'?subject=Series publication'>Your hub's name administrator</a> first."

Notes: You will not see the language if any one of hub's contact email and DOI Publisher is not set, or both of them are not set to non-empty value in publication component configuration on administrator user interface.

No hot fix required.
